pkgs/build-support/fetchpatch/default.nix

```diff
@@ -11,9 +11,13 @@ let
 in
 { stripLen ? 0, extraPrefix ? null, excludes ? [], includes ? [], revert ? false, ... }@args:
 
+let
+  # Make base-64 encoded SRI hash filename-safe using RFC 4648 §5
+  tmpname = lib.replaceStrings [ "+" "/" "=" ] [ "-" "_" "" ] args.sha256;
```

Comment:
why do `+` and `=` need escaping?

Reply:
1. Some file systems do not support those in file names.
2. Wanted to use existing alternative mapping instead of inventing my own.